Sources/Knit/Module/DependencyTree.swift

```diff
@@ -103,18 +102,20 @@ public struct DependencyTree: CustomDebugStringConvertible {
 }
 
 /// Wrapper for the types to allow them to be hashable
-private struct AssemblyReference: Hashable {
+internal struct AssemblyReference: Hashable {
 
     let type: any ModuleAssembly.Type
-    let name: String
+    let id: ObjectIdentifier
+
+    var name: String { String(describing: type) }
```

Comment:
`String(describing: type)` was much slower than `ObjectIdentifier(type)`. This call is only for debugging.

Reply:
Nit: `debugDescription` instead of `name`?